paper/paper.md

```diff
@@ -0,0 +1,105 @@
+
+
+# Related Work
+
+MapReader is among the first end-to-end pipeline for processing historical maps and other images that was designed to lower barriers to experimenting with computer vision in answering research questions about large image datasets. Other projects are emerging which are performing similar research tasks with the visual content in historical map collections `[@Petitpierre; @Combes]`, and of course other tools, like the Distant Viewing Toolkit `[@Arnold]`, address similar needs for other kinds of media. As part of a collaboration between Machines Reading Maps and the David Rumsey Historical Map Collection, the Knowledge Computing Lab released mapKurator `[@mapkurator]`, which takes map image input and creates geojson files of all text on each map.
+
+
+# Documentation
+
+MapReader aims to build computational skills among historians. Our extensive work on documentation and training, including substantial updates to MapReader since `@Hosseini_mapreader`, reflect this commitment. As historians explore the possibilities of computational methods for novel historical research, MapReader models how computational tools can unlock difficult to use primary sources and how we can embrace open research practices as a way to encourage learning. We welcome contributions and requests for new documentation or tutorials.
+
+Our documentation includes:
+- About MapReader: A basic introduction to the software and its origins
+- Events: Activities where the community can engage with MapReader
+- Project CV: Papers, talks, workshops, etc. delivered by the MapReader team
+- Coding Basics: Skills and tools that are useful for getting started with MapReader
+- Installation Instructions: How to install MapReader
+- Input Guidance: What kind of maps and which formats work well in MapReader
+- User Guide: Walkthrough of how to run MapReader
+- Worked Examples: Jupyter notebooks demonstrating uses of MapReader for specific cases (with data, e.g. `@Hosseini_mapreader_data`)
+- API Reference: Auto-generated API reference documentation
+- Code of Conduct and Inclusivity: Our approach to ethical and inclusive conduct
+- Contribution Guide: How to engage with MapReader
+- Developer's Guide: How to update the MapReader version number and upload to package managers
+
+# Conclusion
+
+MapReader enables researchers to ask questions of large collections of maps. It represents a novel approach to digitizing map content, one which intentionally prevents the collection of overly precise data from cartographic documents. MapReader embraces a humanistic approach to data creation and curation, offering an alternative or complement to pixel-level image segmentation.
```

Comment:
Should we add a sentence or two about documentation/tutorials/release here?

Reply:
We have a word limit, but I'll try!